python-squid/src/main/java/org/sonar/python/semantic/SymbolTableBuilderVisitor.java

```diff
@@ -124,6 +124,7 @@ public void visitNode(AstNode node) {
         createScope(node, currentScope);
 
       } else if (node.is(PythonGrammar.EXPRESSION_STMT)) {
+        // TODO PythonGrammar.ANNASSIGN
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.python%3Apython&branch=SONARPY-247&issues=AWDpbCW2GaMZFZs6hksp&open=AWDpbCW2GaMZFZs6hksp)

[ISSUE_KEY]: AWDpbCW2GaMZFZs6hksp